conf/mcp.yaml.template

```diff
@@ -0,0 +1,7 @@
+# For MCP server
```

Comment:
@pondrejk I might suggest being more specific here, as this might not be the only MCP server we'll be using in Robottelo. Let's go with foreman_mcp_server/satellite_mcp_server.  What do you think?